lib/features/cardio_log/presentation/screens/cardio_input_page.dart

```diff
@@ -386,7 +387,6 @@ class CardioInputPageState extends State<CardioInputPage> {
         );
       }
       _navigateAfterSave();
-      
     } catch (e) {
       // Show error message to user
       if (mounted) {
```

Comment:
```suggestion
   } catch (e) {
  // Show error message to user
  if (mounted) {
    ScaffoldMessenger.of(context).showSnackBar(
      SnackBar(content: Text('An error occurred: $e')),
    );
  }
  // Consider logging the error to a monitoring service
  // logError(e);
}
```

Akan lebih apabila ditambahkan sebuah logging error sehingga lebih mudah untuk mentrack issue ata bug